### ISSUE TITLE
Integration documentation needs more detail

### ISSUE THREAD

**Issue by brentc** [django-getpaid/django-plans#31]
I've been over the documentation a few times and I can't find any information on how to actually integrate with django-plans within our views and models or how to integrate our own payment system (without using django-payments).

I've been able to piece some of it together from the example demo, but that's hardly a substitute for actual documentation.

**Comment by Alir3z4** [comment]
That is true, the documentation doesn't properly cover the integration part. At the moment there's only a TODO comment in it https://django-plans.readthedocs.io/en/latest/integration.html#integration

However the demo directory contains most of the stuff needed.
But still, I agree the documentation should do a better job in case of integration.
